{"name": "用户输入运算数据a、b, 程序输出正确的结果", "status": "broken", "statusDetails": {"message": "NameError: name 'cmd_option' is not defined", "trace": "self = <test_case.test_case_01.TestCase object at 0x0000029F846D69F0>, a = 10, b = 5, expected = 2\n\n    @allure.title(\"用户输入运算数据a、b, 程序输出正确的结果\")\n    @allure.testcase(\"http://192.168.40.134:8080/projects/CACULATOR/issues/CACULATOR-4?filter=allissues\",\n                     \"用例管理系统\")\n    @pytest.mark.test_normal\n    @pytest.mark.run(order=1)\n    @pytest.mark.parametrize('a,b,expected', GetData.get_div_data())\n    def test_div(self, a, b, expected):\n        # step2(a, b, expected)\n>       print(cmd_option)\nE       NameError: name 'cmd_option' is not defined\n\ntest_case_01.py:80: NameError"}, "parameters": [{"name": "a", "value": "10"}, {"name": "b", "value": "5"}, {"name": "expected", "value": "2"}], "start": 1715221747198, "stop": 1715221747198, "uuid": "f5bb6ed0-362b-4eb5-92ac-a09d7c4ef048", "historyId": "6449142ca46ed6fcb2e65f95042d23f2", "testCaseId": "7291d55f31158b2c09ba1cb8544f4e8f", "fullName": "test_case.test_case_01.TestCase#test_div", "labels": [{"name": "tag", "value": "run(order=1)"}, {"name": "tag", "value": "test_normal"}, {"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "test_case_01"}, {"name": "subSuite", "value": "TestCase"}, {"name": "host", "value": "wangmin"}, {"name": "thread", "value": "22244-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.test_case_01"}], "links": [{"type": "tms", "url": "http://192.168.40.134:8080/projects/CACULATOR/issues/CACULATOR-4?filter=allissues", "name": "用例管理系统"}]}